libs/driver/magento/src/errors/public_api.ts

```diff
@@ -1 +1,2 @@
 export { daffTransformMagentoError } from './transform';
+export { DaffBadInputError } from './bad-input';
```

Comment:
Seems to me like the `daffTransformMagentoError` should be here is `@daffodil/driver/magento`, but the actual `DaffBadInputError` should be in `@daffodil/driver`

Reply:
hm that's a good point, it does seem like it'll be useful for all platform drivers.

Reply:
moved it